xapian-applications/omega/templates/query

```diff
@@ -108,9 +108,9 @@ $if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesiz
 </div>
 </div>
 </td>
-<td><b><a href="$if{$opt{clicklog},$opt{clicklogurl},$html{$field{url}}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
+<td><b><a href="$html{$opt{clickurl}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
 <small>$snippet{$field{sample}}</small><br>
-<a href="$if{$opt{clicklog},$opt{clicklogurl},$html{$field{url}}}">$html{$field{url}}</a><br>
+<a href="$html{$opt{clickurl}}">$prettyurl{$html{$field{url}}}</a><br>
```

Comment:
`$html` is needed, though really the escaping should be the done **last** (so the outermost operation) - that way it's clear the value is properly escaped (assuming `$html` does its job correctly, which is something that can be tested in isolation).

If you use `$prettyurl{$html{...}}` the reader can't be sure that the output is properly escaped for HTML without looking carefully at what `$prettyurl` does and thinking through how that can interact with HTML-escaped data.

However, you're actually adding a `$prettyurl` which wasn't here before.  It's really a bug that it isn't already used here, and it would be good to fix that bug, but make this change as part of a commit to add clickstream logging is unhelpful.  Each commit should do one thing.  Prettifying the URL here is not connected with clickstream logging, so should be a separate commit.

In particular, adding this missing `$prettyurl` is a candidate for backporting to 1.4.x, but clickstream logging isn't something that's likely to be appropriate to backport.